src/styles/reset.css

```
@@ -0,0 +1,7 @@
+p,
+h2,
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
```

html don't have `margins` or `paddings`
```suggestion
p,
h2,
body {
  margin: 0;
  padding: 0;
}
```